### ISSUE TITLE
Older instances of backup are not removed automatically

### ISSUE THREAD

**Issue by PreetM07** [oTranscribe/oTranscribe#131]
-> I was working on a file and accidently pressed F5 (as it is near to F4!)
I have not closed the browser and was working for whole day. Now if I go into this backup localstorage, it is only showing previous file's backups not today's! I can see one entry as last file... which is having the video info... but rest 99 are trash!!!

Only keep the latest backup of a context.
Remove the older backups and keep saving new (keep overwriting with maybe 5 instances with intervals)
So basically.
1. let's say work on file A is happening...
then keep saving 5-10 instances of its backups. (no need for 100)
2. 11-20 could be file B's backup which was done earlier (last 10 of B)

This way the backup will be shared across 10 files. (may be keep this numbers customizable from settings as per user's wish)

And also maybe give an option to disable the reload of the page!!!!
